gen2-seq-num-sync/requirements.txt

```diff
@@ -0,0 +1,5 @@
+opencv-python==4.5.1.48
+numpy==1.18.3
```

Comment:
numpy version doesn't match the one that comes with opencv 4.5. May cause issues.
`numpy==1.19.5`